examples/chemicalEnvironments/README.md

```diff
@@ -0,0 +1,177 @@
+atom2.addORtype('#7', ['X3', '+0'])
+print("New Atom 2: %s" % atom2.asSMIRKS())
+# New Atom 2: [#8,#7+0X3;R1:2]
+
+print("\nNew SMIRKS: %s" % angle.asSMIRKS())
+# New SMIRKS: [#6X3,#7X2:1]~;@[#8,#7+0X3;R1:2]~;@[#6X3,#7:3]
+```
+
+### Adding and removing atoms
+
+You can add new atoms to an environment by specifying an atom to connect
+to and the OR and AND types for the new bond and new atom. 
+
+You can also remove existing atoms, this move returns True if the atom was removed and False if it wasn't. 
+Removing an atom removes its associated bonds. Calling removeAtom on an indexed atom or an atom connecting two 
+other atoms will always return False. 
+
+```python
+atom3 = angle.selectAtom(3)
+
+alpha_ORtypes = [('#8', ['X2', 'H1'])]
+alpha_ANDtypes = ['R0']
+alpha_bondANDtypes = ['!@']
+alpha = angle.addAtom(atom3, bondANDtypes = alpha_bondANDtypes, 
+                      newORtypes = alpha_ORtypes, newANDtypes = alpha_ANDtypes)
+print("Alpha Atom SMIRKS: %s" % alpha.asSMIRKS())
+# Alpha Atom SMIRKS: [#8X2H1;R0]
+
+beta_ORtypes = [('#1', [])]
+beta = angle.addAtom(alpha, newORtypes = beta_ORtypes)
+print("Beta Atom SMIRKS: %s" % beta.asSMIRKS())
+# Beta Atom SMIRKS: [#1]
+
+print("New overall SMIRKS: %s" % angle.asSMIRKS())
+# New overall SMIRKS: [#6X3,#7X2:1]~;@[#8,#7+0X3;R1:2]~;@[#6X3,#7:3]~;!@[#8X2H1;R0]~[#1]
+
+removed = angle.removeAtom(beta)
+print("The hydrogen beta to atom3 was remove: ", removed)
+# The hydrogen beta to atom3 was remove:  True
+print("Updated SMIRKS string: %s" % angle.asSMIRKS())
+# print("Updated SMIRKS string: %s" % angle.asSMIRKS())
+```
+
+### Other ChemicalEnvironment Methods
+
+0. Selecting atoms and bonds
+```python
+# when given an integer selectAtom or selectBond returns that atom or bond with that index 
+# otherwise it returns a random atom or bond that meets the specified requirement 
+# If no atom or bond with that type is found then None is returned
+alpha = angle.selectAtom('alpha')
+atom4 = angle.selectAtom(4) # None, angle has 3 indexed atoms
+beta_bond = angle.selectBond('beta') # bond between an alpha and beta atom
+```
+
+1. Getting information about an atom or bond in an environment (i.e. isAlpha returns a boolean)
+```python
+# Check if the alpha atom above is any of the following
+angle.isIndexed(alpha) # False
+angle.isUnindexed(alpha) # True 
+angle.isAlpha(alpha) # True
+angle.isBeta(alpha) # False
+# NOTE - These methods can take an atom or a bond as an argument
+```
+2. Get atoms or bonds in each type of position, for example getIndexedAtoms or getAlphaBonds
+```python
+# We will print the SMIRKS for each indexed atom:
+all_atoms = angle.getAtoms()
+indexed_list = angle.getIndexedAtoms() # list of indexed atoms
+unindexed_list = angle.getUnindexedAtoms() # list of atoms with no index
+alpha_list = angle.getAlphaAtoms()
+beta_list = angle.getBetaAtoms
+# Note - Atoms can be replaced with Bonds in each of the above examples
+```
+3. Report the minimum order of a bond with Bond.getOrder
```

Comment:
I'll add more commentary, but this is because there could be more than one bond type. For example, `[*]-,:,=[*]` could be single, aromatic, or double. `getOrder()` on that bond would return 1.